app/views/deenrolment/haveYouStoppedSelfEmployment.scala.html

```diff
@@ -34,7 +34,7 @@
 
         @components.input_radio(
             field = form("value"),
-            legend = components.heading("haveYouStoppedSelfEmployment.heading").body,
+            legend = components.heading("have-you-stopped-sa", "haveYouStoppedSelfEmployment.heading").body,
```

Comment:
is sa self-employment?
Not that it really matters

Reply:
well spotted, I'll change it